src/Lobby.tsx

```diff
@@ -0,0 +1,64 @@
+import type { Socket } from "socket.io-client";
+
+import LobbyForm from "LobbyForm";
+import React, { ReactElement } from "react";
+import { css, StyleSheet } from "aphrodite";
+import logo from "./static/logo.png";
+
+type Props = Readonly<{
+  socket: Socket;
+}>;
+
+function Lobby({ socket }: Props): ReactElement | null {
```

Comment:
doesn't need `| null`, you're never returning null (i know other files do this but just remove it here, we can remove other instances later)